src/name_bazaar/ui/pages/offerings_search_page.cljs

```diff
@@ -16,17 +16,24 @@
     [print.foo :refer [look]]
     [re-frame.core :refer [subscribe dispatch]]
     [reagent.core :as r]
-    [soda-ash.core :as ui]
-    ))
+    [soda-ash.core :as ui]))
 
 (defn offerings-keyword-text-field []
-  (let [search-params (subscribe [:offerings.main-search/params])]
+  (let [search-params (subscribe [:offerings.main-search/params])
+        input-value (r/atom (:name @search-params))
+        debounced-dispatch (debounce
```

Comment:
why not just use `:dispatch-later`?

Reply:
@fbielejec Is there an advantage to using that have over Google Closure's native debounce?

Reply:
Less code, less dependencies, can be guaranteed to play nicely with other reframe events (i.e async-flow).

Reply:
@fbielejec how can you replace a debounce system with :dispatch-later fx? 
Google Closure library is already part of ClojureScript so no extra dep here.
Mabe you are talking about :dispatch-debounce fx in NB? In this case it is a custom implementation of a debounce system.

Reply:
Thats the one, sorry for confusion, I'm typing from the phone.Adding to the argument it uses js/timeout + dispatch, a lot simpler. Lets just not reinvent the wheel and use existing solutions (less code, less maintenance etc) imho.

Reply:
Hehe I'm ok with whatever solution, I thought using the debounce included in cljs was not reinventing the wheel since we don't need to maintain that code. Since the :dispatch-debounce is already there and tested is ok for me.

Reply:
During today's dev meeting Matus asked us to merge this in as is.